_content/animals/can-i-travel-to-the-us-with-dogs.md

```diff
@@ -10,4 +10,4 @@ sources:
 title: Can I travel with or import dogs to the United States during the outbreak?
 ---
 
-Please refer to [CDC’s requirements for bringing a dog to the United States](https://www.cdc.gov/importation/bringing-an-animal-into-the-united-states/index.html). The current [requirements for rabies vaccination](https://www.cdc.gov/importation/bringing-an-animal-into-the-united-states/rabies-vaccine.html) apply to dogs imported from China, a high-risk country for rabies.
\ No newline at end of file
+Please refer to [CDC’s requirements for bringing a dog to the United States](https://www.cdc.gov/importation/bringing-an-animal-into-the-united-states/index.html). The current [requirements for rabies vaccination]( apply to dogs imported from high-risk countries for rabies. The current [requirements for rabies vaccination](https://www.cdc.gov/importation/bringing-an-animal-into-the-united-states/rabies-vaccine.html) apply to dogs imported from high-risk countries for rabies.
```

Comment:
This one is rendering a bit weird. I think you probably don't mean to have the extra ( in there:

> "Please refer to CDC’s requirements for bringing a dog to the United States. The current [requirements for rabies vaccination]**(** apply to"

Reply:
Great catch! Thank you.